calcyoulater.css

```diff
@@ -0,0 +1,55 @@
+
+h1, h2, header {
+    font-family:'Courier New', Courier, monospace;
+}
+
+h1 {
+    text-decoration:underline;
+}
+
+h2 {
+    display:inline-block;
+} 
+
+/* I know, I know - a pretty intensive way to just make a plus sign. :) */
+
+.plussign {
+    background-color: rgb(196, 44, 44);
+    width:20px;
+    height:4px;
+    position: relative;
+    margin:5px 10px;
+    display:inline-block;
+}
+
+.plussign::before {
+    background-color:inherit;
+    position:absolute;
+    width:4px;
+    height:20px;
+    margin-left: 8px;
+    margin-top: -8px;
+    content:'';
+}
```

Comment:
HAHA! Definitely overkill for a real app but great to see you working with pseudo-elements